java/benchmarks/build.gradle

```diff
@@ -12,7 +13,7 @@ dependencies {
     def releaseVersion = System.getenv("GLIDE_RELEASE_VERSION");
 
     if (releaseVersion) {
-        implementation "io.valkey:valkey-glide:" + releaseVersion
+        implementation "io.valkey:valkey-glide:" + releaseVersion + ":${osdetector.classifier}"
```

Comment:
🌟

Reply:
during some point of the testing, the system wasn't able to recognize what os version it was using so that's why we added in the detector for it to work/identify. I can run the CI again without this line to see if it will still work as intended.